src/fivetran-destination/src/destination/ddl.rs

```diff
@@ -109,23 +121,31 @@ pub async fn handle_create_table(request: CreateTableRequest) -> Result<(), OpEr
         }
     }
 
-    // TODO(benesch): support primary keys.
-    #[allow(clippy::overly_complex_bool_expr)]
-    if !primary_key_columns.is_empty() && false {
-        defs.push(format!("PRIMARY KEY ({})", primary_key_columns.join(",")));
-    }
-
     let sql = format!(
         r#"BEGIN; CREATE SCHEMA IF NOT EXISTS {schema}; COMMIT;
-           BEGIN; CREATE TABLE {schema}.{table} ({defs}); COMMIT;"#,
-        schema = escape::escape_identifier(&request.schema_name),
-        table = escape::escape_identifier(&table.name),
+        BEGIN; CREATE TABLE {qualified_table_name} ({defs}); COMMIT;"#,
         defs = defs.join(","),
     );
 
     let (_dbname, client) = config::connect(request.configuration).await?;
     client.batch_execute(&sql).await?;
 
+    // TODO(parkmycar): This is an ugly hack!
+    //
+    // If Fivetran creates a table with primary keys, it expects a DescribeTableRequest to report
+    // those columns as primary keys. But Materialize doesn't support primary keys, so we need to
+    // store this metadata somewhere else. For now we do it in a COMMENT.
+    for column_name in primary_key_columns {
+        let stmt = format!(
+            "COMMENT ON COLUMN {qualified_table_name}.{column_name} IS {magic_comment}",
```

Comment:
Also make this a param.

Reply:
`COMMENT ON` doesn't support specifying the comment with a param, it needs to be a string literal. This is also the case in Postgres, e.g. the following fails:
```
let mut client = postgres::Client::connect("host=localhost user=parker.timmerman", NoTls)?;

client.execute("CREATE TABLE t1 (a int)", &[])?;
client.execute("COMMENT ON COLUMN t1.a IS $1", &[&"hello world"])?;
```